modules/services/pass-secret-service.nix

```diff
@@ -2,31 +2,47 @@
 
 with lib;
 
-let serviceCfg = config.services.pass-secret-service;
+let cfg = config.services.pass-secret-service;
 in {
   meta.maintainers = [ maintainers.cab404 ];
+
   options.services.pass-secret-service = {
     enable = mkEnableOption "Pass libsecret service";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.pass-secret-service;
+      defaultText = literalExpression "pkgs.pass-secret-service";
+      description = "Package providing <command>pass_secret_service</command>.";
+    };
+
+    storePath = mkOption {
+      type = types.str;
+      default = "$PASSWORD_STORE_DIR";
+      defaultText = "$PASSWORD_STORE_DIR";
```

Comment:
No need to put a `defaultText` attribute since the value is a plain string.
```suggestion
```

Reply:
Fixed in c211c509